tests/phpunit/tests/option/networkOption.php

```diff
@@ -794,60 +794,37 @@ public function test_update_network_option_should_conditionally_apply_site_and_o
 	}
 
 	/**
-	 * Tests that update_network_option() adds a non-existent option that uses a filtered default value.
+	 * Tests that update_network_option() does not add a non-existent option that uses a filtered default value in multisite.
 	 *
 	 * @ticket 59360
 	 *
 	 * @covers ::update_network_option
 	 */
-	public function test_update_network_option_should_add_option_with_filtered_default_value() {
+	public function test_update_network_option_should_not_add_option_with_filtered_default_value() {
 		global $wpdb;
 
-		$option               = 'foo';
-		$default_site_value   = 'default-site-value';
-		$default_option_value = 'default-option-value';
+		if ( ! is_multisite() ) {
+			$this->markTestSkipped( 'This test only runs in Multisite.' );
+		}
```

Comment:
Keeping this intentionally just in case the test is reached in single site mode.

Reply:
Initially my thinking that keeping a test for adding an option with a filtered default value on single site might be cutting a bit too close to testing the behaviour of `update_option()`, but now that I think about it, my thinking was too coupled to implementation. The bottom line is that if `update_network_option()` is called on single site for an non-existent option with a filtered default value and a new value of `false`, it should be added to the database with a value of `false` (read: `(string) ""`)

Rather than merge the logic with conditionals, I'll add a separate test for single site so it's clear that each is testing for a different outcome.

Nice catch!

Reply:
Added in [e59f9d2](https://github.com/WordPress/wordpress-develop/pull/5465/commits/e59f9d2b7ff98242c341a37de259c55985734946).